.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+closes #
+
+---
+
+Submission checklist:
+
+<!-- Remove anything below that is not applicable -->
+
```

Comment:
Either uncomment this line or remove, because it's not visible. LGTM otherwise.
I think the documentation is also a bit overkill. Like we never have doc :dancers:

Reply:
@Tbaut ok, removed :)